docs/content.zh/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md

```diff
@@ -50,12 +50,12 @@ using a build automation tool (such as Maven or SBT) and SQL Client with SQL JAR
 
 ```Download link is available only for stable releases.```
```

Comment:
It seems redundant now, there is no unavailable link now.

Reply:
addressed it.